test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+
+npm run lint
+
+function cleanup() {
+	docker stop deltatest || true
+	docker rm --volumes deltatest || true
+}
+trap cleanup EXIT
+
+docker build -f Dockerfile.test -t docker-delta-test .
+docker stop deltatest || true
+docker rm --volumes deltatest || true
+docker run --privileged --name deltatest -d docker-delta-test
```

Comment:
Instead of `--privileged` you can run with the following (though I'm not sure whether this is more constraining than `--privileged`):

```
docker run \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_ADMIN \
  --cap-add=SYS_RESOURCE \
  --security-opt apparmor=unconfined \
  --security-opt seccomp=unconfined \
  --name deltatest -d docker-delta-test
```